06-walkthrough-1.Rmd

```diff
@@ -348,12 +654,59 @@ m_course <- lmer(percentage_earned ~ TimeSpent + (1|course_id), data = dat)
 summary(m_course)
 ```
 
-A common way to understand how much variability is at the group level is to calculate the *intra-class* correlation. This value is the proportion of the variability in the outcome (the *y*-variable) that is accounted for solely by the groups identified in the model. There is a useful function in the **sjstats** package for doing this.
+As we mentioned earlier, a common way to understand how much variability is at 
+the group level is to calculate the *intra-class* correlation. This value is the
+proportion of the variability in the outcome (the *y*-variable) that is 
+accounted for solely by the groups identified in the model. There is a useful 
+function in the **performance** package for doing this.
 
 ```{r}
-# install.packages("sjstats")
-library(sjstats)
+# install.packages("performance")
+library(performance)
 icc(m_course)
 ```
 
-This shows that nearly 17% of the variability in the percentage of points students earned can be explained simply by knowing what class they are in. 
+This shows that nearly 17% of the variability in the percentage of points 
+students earned can be explained simply by knowing what class they are in. 
+
+There is much more to do with multi-level models. We briefly discuss a 
+common extension to the model we just used - although such a model, in which
+the effects of being a student in a class are recognized through specifying 
+what the groups are, is very common. The extensions is for adding 
+additional levels.
```

Comment:
I'm a little confused by the paragraph from 672-676. Which is common: the one we just did, or the one we will explain below? I think that paragraph could use some edits for clarity